### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter February 2021

### PR DESCRIPTION
**Pull request by Echolon**
### Final release is planned for **5th of March 2021**.
---
You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (commteam@muc.xmpp.org)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-january2021/content/posts/newsletter/2021-02-05.md)

:arrow_right: Everyone participating, please do not just drop bare links - rather try to make a short and decent formulated description (~5 lines) what is the main outcome of your source. You do not need to write formal - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

---

This is our planed timeline for this draft:

XMPP Newsletter Timeline

Status:
- [x] Collection of news
- [x] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/posts/newsletter/2021-03-05.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released with some delay here:
+
+- French on [linuxfr.org](https://linuxfr.org/tags/xmpp/public) and [jabberfr.org](https://news/jabberfr.org/category/newsletter/).
+- The German translation of the [XMPP Newsletter for January 2021 is available at Anoxinon](https://anoxinon.de/blog/xmpp_newsletter_januar_2021/) [DE].
+
+## XSF Announcement
+
+GSOC
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-03-05.md:30]: Placeholder missing some explanation ;)

File: content/posts/newsletter/2021-03-05.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+[Kaidan 0.7 has been released](https://www.kaidan.im/2021/02/02/kaidan-0.7.0/): This release enables users to send files via drag and drop. Furthermore, it is now possible to see which chat client and operating system a contact is using.
+
+[Monal 5 Beta 1 has been released](https://monal.im/blog/monal-5-beta-1/), which brings many improvements with OMEMO. While in Beta, the Monal developers are working on support for MUC, and a new file transfer user interface. Monal 5 will also support [inline display of audio and video files](https://monal.im/blog/monal-5-will-support-inline-audio-and-video/). [Push server stats from January](https://monal.im/blog/monal-stats-1-2021/) reveal quite some activity around Monal.
+
+ADD MISSING MONAL NEWS
```
Comment (SamWhited) [inline content/posts/newsletter/2021-03-05.md:74]: Needs fixing or removing

File: content/posts/newsletter/2021-03-05.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+ 
+The PeerTube plugin livechat [ships with Converse.js](https://github.com/JohnXLivingston/peertube-plugin-livechat).
+ 
+[SiskinIM 6.3 has been released](https://github.com/tigase/siskin-im/releases/tag/6.3) containing several fixes (clicking on links, crashes related to markdown formatting, sharing in public groups, issues with message corrections and groupchat layout improvements).
+ 
+## Servers
+
+[Snikket’s February 2021 Server Release](https://snikket.org/blog/feb-2021-server-release/) brings a fancy new web-based admin dashboard, allowing you to perform all kinds of management tasks. It also adds a simple portal for non-administrative users to log in and manage their account. Currently this contains a profile editor, and will grow to include other account settings, and data import/export. And there is more: Raspberry Pi and ARM support, and the introduction of user groups, known as "circles".
+
+[IMAGE SNIKKET]
```
Comment (SamWhited) [inline content/posts/newsletter/2021-03-05.md:86]: Needs an actual image?

**Comment by Echolon** [comment]
### :hourglass:  **Editorial closing of the XMPP Newsletter February release** 

### :rocket: Publishing in a few minutes!

### :pray: Thanks to wurstsalat3000, Sam Whited, pitchum, Licaon_Kter, jeybe and emus for their help in creating it!
### :handshake:   ... ehh.. here we go again, please let it end... :hand: :mask: :hand: 

### :heart: :fireworks: Next release on 5th April 2021

**Pull request merged**